composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "hypothesis/hypothesis",
+    "name": "hypothesis/hypothesis-wp-plugin",
```

Comment:
I'm not sure if this should be `hypothesis/wp-hypothesis` to match the repository name 🤔

Reply:
It will make it easier to find the repo if it has the same name as the plugin.

Reply:
Yep, agree